freqtrade/configuration/configuration.py

```diff
@@ -462,6 +462,9 @@ def _process_analyze_options(self, config: Config) -> None:
         self._args_to_config(config, argname='indicator_list',
                              logstring='Analysis indicator list: {}')
 
+        self._args_to_config(config, argname='timerange',
+                             logstring='Filter trades by timerange: {}')
+
```

Comment:
The timerange parameter is already present - no need to add it again - it'll cause duplicated log messages...) - as well as having no reason